src/main/java/com/commercetools/sync/products/templates/beforeupdatecallback/SyncSingleLocale.java

```diff
@@ -41,13 +42,14 @@
         @Nonnull final ProductDraft newProductDraft,
         @Nonnull final Product oldProduct,
         @Nonnull final ProductType productType) {
-        return updateActions.stream()
-                            .map(action ->
-                                filterSingleLocalization(action, newProductDraft, oldProduct, productType,
-                                    Locale.FRENCH))
-                            .filter(Optional::isPresent)
-                            .map(Optional::get)
-                            .collect(toList());
+
+        final List<Optional<UpdateAction<Product>>> optionalActions =
+            updateActions.stream()
+                         .map(action -> filterSingleLocalization(action, newProductDraft, oldProduct, productType,
+                             Locale.FRENCH))
```

Comment:
why is Locale.FRENCH hardcoded? (when you need a default  value I would suggest to use `Locale.English
`)

Reply:
because if you see the name of the method `syncFrenchDataOnly` and the `Javadoc`. It is meant to showcase that exact scenario with syncing a single french locale.